packages/db/src/schema.ts

```diff
@@ -744,6 +760,12 @@ export const apExamReward = pgTable("ap_exam_reward", {
   coursesGranted: json("courses_granted").$type<APCoursesGrantedTree>().notNull(),
 });
 
+export const sampleProgram = pgTable("sample_program", {
+  programName: varchar("program_name").notNull(),
+  sampleProgram: json("sample_program").$type<SampleProgramEntry[]>().notNull(),
+  programNotes: varchar("program_notes").notNull(),
```

Comment:
if you intend to store a list of strings, use an array of varchar

this will also obviate the JSON parsing code in the endpoint layer